docs/metrics/metrics-operators/min.md

```diff
@@ -18,17 +18,17 @@ min [by FIELD [, FIELD, ...]]
 
 ### Minimum metric value across all matching time series
 
-This query returns the minimum  `cpu_system` value for each time interval across all matching time series.
+This query returns the minimum `CPU_Sys` value for each time interval across all matching time series.
 
 ```sql
-dep=prod metric=cpu_system | min
+dep=prod metric=CPU_Sys | min
 ```
 
 ### Minimum metric value across by a tag value
 
-This query returns the minimum cpu_system value by node for each time interval across all matching time series.
+This query returns the minimum `CPU_System` value by node for each time interval across all matching time series.
```

Comment:
I would suggest to be consistent with `CPU_Sys` and ``CPU_System` as well

Reply:
Ops, missed this one. Corrected.